src/components/containers/notification/Notification.js

```diff
@@ -146,6 +146,13 @@ function renderCallsToAction(callsToAction) {
       {callsToAction.map((callToAction, index) => {
         const buttonStyleType = index === 0 ? 'primary' : 'default';
 
+        if (React.isValidElement(callToAction)) {
```

Comment:
Can you follow up on getting that done sometime @aabenoja? Other than that, I'm OK with this as a stop-gap solution for the problem at hand, but we should definitely give it some more thought.